src/theming/Components/Preview.tsx

```diff
@@ -0,0 +1,38 @@
+import {
+  Box,
+  Button,
+  Heading,
+  Intent,
+  Scheme,
+  toThemeClass
+} from "@contember/ui"
+import * as React from 'react'
+
+export const Preview = ({
+  heading,
+  scheme,
+  theme,
+  themeContent,
+  themeControls,
+}: {
+  heading: string,
+  scheme: Scheme,
+  theme?: Intent,
+  themeContent?: Intent,
+  themeControls?: Intent,
+}) => <Box
+  gap="default"
+  className={[
+    `scheme-${scheme}`,
+    toThemeClass(themeContent ?? theme, themeContent ?? theme)
+  ].join(' ')}
+>
+  <Heading>{heading}</Heading>
+  <Button onClick={React.useCallback(() => { alert('Hi!') }, [])}>Click me!</Button>
```

Comment:
Do not use `alert()`.

```suggestion
  <Button>Default button</Button>
```